### PR TITLE
Add Read the documentation links to each paragraph

### PR DESCRIPTION
**Pull request by peter-gergely-horvath**
This fixes https://github.com/apache/airflow/issues/16431

### CODE REVIEW

File: landing-pages/site/content/en/docs/_index.md [hunk 1]
```diff
@@ -157,14 +159,14 @@ Providers packages include integrations with third party integrations. They are
 
 Airflow has an official Dockerfile and Docker image published in DockerHub as a convenience package for
 installation. You can extend and customize the image according to your requirements and use it in
-your own deployments.
-
+your own deployments.
```
Comment (potiuk) [inline landing-pages/site/content/en/docs/_index.md:162]:
```suggestion
your own deployments.
```

File: landing-pages/site/content/en/docs/_index.md [hunk 2]
```diff
@@ -157,14 +159,14 @@ Providers packages include integrations with third party integrations. They are
 
 Airflow has an official Dockerfile and Docker image published in DockerHub as a convenience package for
 installation. You can extend and customize the image according to your requirements and use it in
-your own deployments.
-
+your own deployments.  
+[Read the documentation >>](/docs/docker-stack/index.html)
```
Comment (potiuk) [inline landing-pages/site/content/en/docs/_index.md:163]:
```suggestion
[Read the documentation >>](/docs/docker-stack/index.html)
```

File: landing-pages/site/content/en/docs/_index.md [hunk 3]
```diff
@@ -157,14 +159,14 @@ Providers packages include integrations with third party integrations. They are
...
-
+your own deployments.  
+[Read the documentation >>](/docs/docker-stack/index.html)  
 
 
 ## [Helm Chart](/docs/helm-chart/stable/index.html)
 
-Airflow has an official Helm Chart that will help you set up your own Airflow on a cloud/on-prem Kubernetes environment and leverage its scalable nature to support a large group of users. Thanks to Kubernetes, we are not tied to a specific cloud provider.
-
+Airflow has an official Helm Chart that will help you set up your own Airflow on a cloud/on-prem Kubernetes environment and leverage its scalable nature to support a large group of users. Thanks to Kubernetes, we are not tied to a specific cloud provider.
```
Comment (potiuk) [inline landing-pages/site/content/en/docs/_index.md:168]:
```suggestion
Airflow has an official Helm Chart that will help you set up your own Airflow on a cloud/on-prem Kubernetes environment and leverage its scalable nature to support a large group of users. Thanks to Kubernetes, we are not tied to a specific cloud provider.
```

**Comment by potiuk** [comment]
small whitespace issues

**Review by kaxil — APPROVED** [review]

**Pull request merged**